weave

```diff
@@ -1634,6 +1634,7 @@ case "$COMMAND" in
         protect_against_docker_hang
         VOLUME_CONTAINERS=$(docker ps -qa --filter label=weavevolumes)
         [ -n "$VOLUME_CONTAINERS" ] && docker rm -v $VOLUME_CONTAINERS  >/dev/null 2>&1 || true
+        rm -f $HOST_ROOT/var/lib/weave/weave-netdata.db >/dev/null 2>&1 || true
```

Comment:
On my end I have, like Martynas: 
[1] `/var/lib/weave` + 
[2] `/weavedb/weave-net` + 
[3] `data.db` = 
`/var/lib/weave/weavedb/weave-netdata.db` != 
`/var/lib/weave/weave-netdata.db`

so it would seem the path is missing `weavedb` (unless I've missed something myself!)

[1] `hostPath: path: /var/lib/weave` from https://github.com/weaveworks/weave/pull/2967/files#diff-63ba20698025213b0dce5f67c19f76dbR106

[2] `--db-prefix="/weavedb/weave-net"` from https://github.com/weaveworks/weave/pull/2967/files#diff-eb27072d080b69e928812b51e99c8f9dR120
and then from `&dbPrefix, []string{"-db-prefix"}, "/weavedb/weave"`:
- https://github.com/weaveworks/weave/blob/master/prog/weaver/main.go#L202
- https://github.com/weaveworks/weave/blob/master/prog/weaver/main.go#L320
- https://github.com/weaveworks/weave/blob/master/db/boltdb.go#L37
- https://github.com/weaveworks/weave/blob/master/db/boltdb.go#L33

[3] `fileName  = "data.db"` from: 
- https://github.com/weaveworks/weave/blob/master/db/boltdb.go#L33
- https://github.com/weaveworks/weave/blob/master/db/boltdb.go#L29